src/include/distributed/coordinator_protocol.h

```diff
@@ -83,6 +83,31 @@ typedef enum
 	SHARD_PLACEMENT_RANDOM = 3
 } ShardPlacementPolicyType;
 
+typedef enum TableDDLCommandType
+{
+	TABLE_DDL_COMMAND_STRING,
+} TableDDLCommandType;
+
+
+typedef struct TableDDLCommand
```

Comment:
These structs could use some comments describing what they are for.

I suppose it may be worth noting that this is not strictly for "DDL". (I think the name is good enough, but comment would be helpful)